src/accelerate/commands/estimate.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/env python
+
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+
+from requests.exceptions import HTTPError
+
+from accelerate import init_empty_weights
+from accelerate.utils import (
+    calculate_maximum_sizes,
+    convert_bytes,
+    is_huggingface_hub_available,
+    is_timm_available,
+    is_transformers_available,
+)
+
+
+if is_huggingface_hub_available():
+    from huggingface_hub import HfApi
```

Comment:
Let's just import `model_info` since you use `HfApi` without changing the defaults.